### PR TITLE
GTEST/UCT/CMA: Fix CMA peer failure test

### LINKED ISSUE #6767: failure on cma/test_ucp_peer_failure_rma_zcopy.put
```
[----------] 1 test from cma/test_ucp_peer_failure_rma_zcopy
[ RUN      ] cma/test_ucp_peer_failure_rma_zcopy.put/0 <cma/memory>
/scrap/azure/agent-02/AZP_WORKSPACE/1/s/contrib/../test/gtest/uct/test_peer_failure.cc:650: Failure
Expected: (m_err_count) > (0ul), actual: 0 vs 0
[  FAILED  ] cma/test_ucp_peer_failure_rma_zcopy.put/0, where GetParam() = cma/memory (0 ms)
```

https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=17628&view=logs&j=da813497-1cca-54a8-f0a9-cb3fd519bc00&t=7dd3ffab-efd6-51e7-77f1-40598eb2014b

HOSTNAME=swx-rdmz-ucx-gpu-01

### PR DESCRIPTION
**Pull request by brminich**
## What
fixes #6767

### CODE REVIEW

**Review by hoopoepg — APPROVED** [review]

File: test/gtest/uct/test_peer_failure.cc [hunk 1]
```diff
@@ -627,20 +627,17 @@ class test_ucp_peer_failure_rma_zcopy : public test_uct_peer_failure
...
-                                          &m_dummy_comp);
+                                          UINT64_MAX, 0ul, &m_dummy_comp);
             } else {
                 status = uct_ep_get_zcopy(m_sender->ep(0),
                                           sendbuf.iov(), 1,
-                                          0xDEADBEAF, 0ul,
-                                          &m_dummy_comp);
+                                          UINT64_MAX, 0ul, &m_dummy_comp);
             }
             EXPECT_FALSE(UCS_STATUS_IS_ERR(status)) << ucs_status_string(status) ;
```
Comment (dmitrygx) [inline test/gtest/uct/test_peer_failure.cc:645]: minor: remove whitespace before semicolon

**Review by dmitrygx — APPROVED** [review]

File: test/gtest/uct/test_peer_failure.cc [hunk 2]
```diff
@@ -627,20 +627,17 @@ class test_ucp_peer_failure_rma_zcopy : public test_uct_peer_failure
...
-            mapped_buffer recvbuf(size, 0, *m_receivers.front());
             ucs_status_t status;
 
             // Pretend peer failure by using invalid address
             if (is_put_op) {
                 status = uct_ep_put_zcopy(m_sender->ep(0),
                                           sendbuf.iov(), 1,
-                                          0xDEADBEAF, 0ul,
-                                          &m_dummy_comp);
+                                          UINT64_MAX, 0ul, &m_dummy_comp);
```
Comment (yosefe) [inline test/gtest/uct/test_peer_failure.cc:636]: minor: maybe define static const uint64_4 INVALID_ADDR = UINT64_MAX;
to document the reason we pass it

**Review by yosefe — APPROVED** [review]

File: test/gtest/uct/test_peer_failure.cc (hunk 1 above)
Comment (brminich) [reply test/gtest/uct/test_peer_failure.cc:645]: fixed

File: test/gtest/uct/test_peer_failure.cc (hunk 2 above)
Comment (brminich) [reply test/gtest/uct/test_peer_failure.cc]: fixed

**Review by dmitrygx — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Pull request merged**